samples/cpp/benchmark_app/main.cpp

```diff
@@ -414,6 +414,17 @@ int main(int argc, char* argv[]) {
                 set_infer_precision();
             } else if (device.find("AUTO") != std::string::npos) {
                 device_nstreams.erase(device);
+            } else if (device.find("MULTI") != std::string::npos) {
+                setThroughputStreams();
+                set_infer_precision();
+                if ((device_name.find("GPU") != std::string::npos) && (device_name.find("CPU") != std::string::npos)) {
+                    slog::warn << "GPU throttling is turned on. Multi-device execution with "
+                                  "the CPU + GPU performs best with GPU throttling hint, "
+                               << "which releases another CPU thread (that is otherwise "
+                                  "used by the GPU driver for active polling)."
+                               << slog::endl;
+                    device_config[GPU_CONFIG_KEY(PLUGIN_THROTTLE)] = "1";
```

Comment:
use API 2.0 property and with device::properties

Reply:
@wangleis since these 2 properties are set to MULTI:GPU,CPU in benchmark_app by default, in this case MULTI will not go to the by default TPUT.